shared/reducers/teams.tsx

```diff
@@ -91,7 +91,16 @@ export default Container.makeReducer<
     details.settings = action.payload.settings
     details.invites = new Set(action.payload.invites)
     details.subteams = action.payload.subteamIDs
-    details.requests = new Set(action.payload.requests.get(teamname))
+
+    const requests = action.payload.requests.get(teamname)
```

Comment:
this reducer case is gone in master. See `constants/teams#annotatedTeamToDetails`

Reply:
at this point i might as well repurpose this branch for the next ticket